stable/minibroker/imagelist.txt

```diff
@@ -1 +1,2 @@
-minibroker:1b8db9db6bd9b20448599d81dee0ebcd896c1c43
+registry.suse.com/cap-staging/minibroker:1.1.0
```

Comment:
This is wrong - the correct repository should be:
```registry.suse.com/cap/minibroker```

Also, there's a duplicated entry.

Reply:
On second thought, not sure how necessary is this `imagelist.txt` file.

Reply:
Ping @dragonchaser.

Reply:
Can we manually update this for now until we fix the release tool?

Reply:
Manually deleted for now. I'll file a bug to look into what the release tool is doing here.